src/yz_events.erl

```diff
@@ -59,7 +70,26 @@
 %%%===================================================================
 
 start_link() ->
-    gen_server:start_link({local, ?MODULE}, ?MODULE, [], []).
+    gen_event:start_link({local, ?MODULE}).
+
+add_handler(Handler, Args) ->
+    gen_event:add_handler(?MODULE, Handler, Args).
+
+add_sup_handler(Handler, Args) ->
+    gen_event:add_sup_handler(?MODULE, Handler, Args).
+
+add_guarded_handler(Handler, Args) ->
+    yz_eventhandler_sup:start_guarded_handler(?MODULE, Handler, Args).
```

Comment:
We may be able too. I don't remember why I didn't initially. I def. thought about and am wondering if there was something too specific related to core w/ it, but maybe not. Sadly, been too long for me to remember.